src/main/java/tr/com/srdc/cda2fhir/jolt/MedicationMap.java

```diff
@@ -0,0 +1,188 @@
+package tr.com.srdc.cda2fhir.jolt;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import tr.com.srdc.cda2fhir.conf.Config;
+import tr.com.srdc.cda2fhir.transform.util.impl.IdentifierMap;
+
+public class MedicationMap {
+
+	Map<String, MedicationContainer> ccMap = new HashMap<String, MedicationContainer>();
+
+	public class MedicationContainer {
+
```

Comment:
Is this really intended to be inner class? Inner classes has uses but it looks like this should be "static". I have not used inner classes for a while but last time I used them they introduce subtle bugs.

Reply:
Yeah I think static sounds right. Not sure what bugs inner classes could cause. I've never had issues with them.